Libplanet.Headless/Hosting/LibplanetNodeService.cs

```diff
@@ -363,6 +363,14 @@ Task BootstrapSwarmAsync(int depth)
 
                         Log.Error(e, "{Message}", e.Message);
                     }
+                    catch (Exception e)
```

Comment:
It can't catch from [rethrowing](https://github.com/planetarium/NineChronicles.Headless/pull/464/files#diff-99583c6ac600be724a6dcad7bad852f486e53190003ccaa7f17afc153b80b81bL361). we need to nest try-catch block if want to catch exceptions from there.